airflow-core/src/airflow/ui/src/i18n/README.md

```diff
@@ -17,8 +17,221 @@
+
+- Before merging any translation-related PR, it MUST be:
+   - Approved by a translation owner of the respective locale for language aspects, according to the standards and guidelines.
+     - When a translation owner initiates a PR and is the only one assigned to the locale, they SHOULD instead ask for approval from a third party (e.g., engaged translator), or if such is not available, declare their self-approval for the language aspects.
+   - Approved by a code owner, or another committer on their behalf, for technical aspects (e.g., linting, formatting, etc.).
+- Before merging a translation-related PR, the translation SHOULD be checked for completeness using the provided tools (see section 8).
+
+> [!WARNING]
+> In languages with different word order than English, or in Right-To-Left (RTL) languages, it is important to validate that the changes are properly reflected in the UI.
+> If they are not, please raise a GitHub issue or a PR for fixing it (separately from the translation PR).
+
+### 5.8. Version release
+
+- Release managers MUST follow the requirements for releasing changes in supported locales defined in the [Release Management Policy](../../../../../../dev/README_RELEASE_AIRFLOW.md).
+
+## 6. Procedures
+
+### 6.1. Approval of ownership candidates
+
+- The designated code owner, should post a thread to the dev list, requesting the approval of:
+  - Themselves as the code owner in the suggested locale. If they are a translation sponsor, they should indicate it as well.
+  - Other code owner(s) in the suggested locale, if applicable.
+  - The identity of the translation owner(s) in the suggested locale (which could be themselves as well)
+- When introducing a new locale, the thread should announce that (including a link to the PR)
+- Within the thread, the code owner should demonstrate that the translation owner is suitable for the role, according to the requirements in section 5.3.
+- Approval of a code owner who is also the translation owner can be done by a lazy consensus.
+- Approval of any translation owner who is not a committer requires at least one binding vote of 1 PMC member, and no objections from other committers/PMC.
+
+### 6.2. Approval of a new locale
+
+The following steps outline the process for approving a new locale to be added to the supported locales:
+
+- Creating a PR for adding the suggested locale to the codebase ([see example](https://github.com/apache/airflow/pull/51258/files)), which includes:
+    - The locale files (translated according to the guidelines) in the `airflow-core/src/airflow/ui/src/i18n/locales/<LOCALE_CODE>` directory, where `<LOCALE_CODE>` is the code of the language according to ISO 639-1 standard (e.g., `fr` for French). Languages with regional variants should be handled in separate directories, where the name is suffixed with `-<VARIANT>`, and `<VARIANT>` is the variant follows ISO 3166-1 or UN M.49 codes in lowercase (e.g., `zh-tw` for Taiwanese Chinese).
```

Comment:
```suggestion
    - The locale files (translated according to the guidelines) in the `airflow-core/src/airflow/ui/src/i18n/locales/<LOCALE_CODE>` directory, where `<LOCALE_CODE>` is the code of the language according to ISO 639-1 standard (e.g., `fr` for French). Languages with regional variants should be handled in separate directories, where the name is suffixed with `-<VARIANT>`, and `<VARIANT>` is the variant that follows ISO 3166-1 or UN M.49 codes in lowercase (e.g., `zh-tw` for Taiwanese Mandarin).
```

I re-read the policy and fixed my comment. I also noticed communities that use  IETF language tag  (BCP 47) with ISO 639-1